tests/unit/test_main.py

```diff
@@ -1016,3 +1016,670 @@ def test_match_and_apply(self, flow, args, resp):
+                        }, {
+                            "action_type": "output",
+                            "port": 25
+                        }]}
+                    ],
+                    "table_id": 0,
+                    "table_group": "evpl",
+                    "priority": 20100,
+                    }},
+                {"flow": {
+                    "match": {
+                        "in_port": 22,
+                        "dl_vlan": 100
+                        },
+                    "instructions": [{
+                        "instruction_type": "apply_actions",
+                        "actions": [{
+                            "action_type": "add_int_metadata"
+                        }, {
+                            "action_type": "set_vlan",
+                            "vlan_id": 100
+                        }, {
+                            "action_type": "push_vlan",
+                            "tag_type": "s"
+                        }, {
+                            "action_type": "set_vlan",
+                            "vlan_id": 1
+                        }, {
+                            "action_type": "output",
+                            "port": 11
+                        }]}
+                    ],
+                    "table_id": 2,
+                    "table_group": "base",
+                    "priority": 20000,
+                    }},
+                {"flow": {
+                    "match": {
+                        "in_port": 26,
+                        "dl_vlan": 1
+                        },
+                    "instructions": [{
+                        "instruction_type": "apply_actions",
+                        "actions": [{
+                            "action_type": "send_report"
+                        }]},
+                        {
+                            "instruction_type": "goto_table",
+                            "table_id": 2
+                        }
+                    ],
+                    "table_id": 0,
+                    "table_group": "evpl",
+                    "priority": 20000,
+                    }, },
+                {"flow": {
+                    "match": {
+                        "in_port": 26,
+                        "dl_vlan": 1
+                        },
+                    "instructions": [{
+                        "instruction_type": "apply_actions",
+                        "actions": [{
+                            "action_type": "pop_int"
+                        }, {
+                            "action_type": "pop_vlan"
+                        }, {
+                            "action_type": "output",
+                            "port": 22
+                        }]}
+                    ],
+                    "table_id": 2,
+                    "table_group": "base",
+                    "priority": 20000,
+                    }, }
+            ]
+        }
+
+        payload = [
+            {"trace": {
```

Comment:
Excellent test cov for `telemetry_int` inter evpl flows. 

Could you also cover another payload starting on sw2 port 22 with vlan 100?

Reply:
Ok.